### ISSUE TITLE
Rocket launcher backblast launches humans back and burns them.

### ISSUE THREAD

**Issue by lostpreviousaccoun** [studio-minus/ppg-bugs-and-requests#1465]
**Feature description**
When firing a rocket launcher, The backblast should launch humans back while burning the affected area.
(optional) If the backflast is close enough to a wall, The gas bounces off and hits the rpg/human holding it instead, Burning and launching both.

**Feature purpose**
Pain and suffering.

**Additional context**
https://www.youtube.com/watch?v=BCN0y4BHiGs&t=32s 12 seconds in. 
The backblast here doesn't kill him but I don't want to force you to see a bloody puddle on the floor with the only things left being limbs a rifle and dogtags.

**Comment by unknown-4o4** [comment]
Yikes, I don't think the video is necessary for such a simple request.

**Comment by DindinYT37** [comment]
> **Feature description**
> When firing a rocket launcher, The backblast should launch humans back while burning the affected area.
> (optional) If the backflast is close enough to a wall, The gas bounces off and hits the rpg/human holding it instead, Burning and launching both.
> 
> **Feature purpose**
> Pain and suffering.
> 
> **Additional context**
> 12 seconds in. 
> The backblast here doesn't kill him, but I don't want to force you to see a bloody puddle on the floor with the only things left being limbs a rifle and dogtags.

Wtf? Blood and gore is not allowed in any way.

**Comment by ppgplayer** [comment]
There is no blood OR gore in that video 

Did you idiots actually click the link?

**Comment by DindinYT37** [comment]
> There is no blood OR gore in that video
> 
> Did you idiots actually click the link?

No, and now guess why. Even if there isn't, why would we click if there could potentially be - you're actually the only idiot present here thinking we'd just be going around clicking links and watching videos from a random person saying something about a "bloody puddle".

**Comment by unknown-4o4** [comment]
> > There is no blood OR gore in that video
> > Did you idiots actually click the link?
> 
> No, and now guess why. Even if there isn't, why would we click if there could potentially be - you're actually the only idiot present here thinking we'd just be going around clicking links and watching videos from a random person saying something about a "bloody puddle".

Exactly
